app/rifcs/rif_cs_generator.rb

```diff
@@ -130,4 +138,12 @@ def build_rif_cs
     end
   end
 
+  private
+
+  def get_title(url)
```

Comment:
Use the wrapper_object to do this - take a look at package_rif_cs_wrapper.rb - that is where this code belongs.